daisy_workflows/image_import/windows/translate_windows_81_byol.wf.json

```diff
@@ -0,0 +1,74 @@
+{
+  "Name": "translate-windows-81-byol",
+  "Vars": {
+    "source_disk": {
+      "Required": true,
+      "Description": "The Windows 8.1 GCE disk to translate."
+    },
+    "install_gce_packages": {
+      "Value": "true",
+      "Description": "Whether to install GCE packages."
+    },
+    "sysprep": {
+      "Value": "false",
+      "Description": "Whether to run sysprep before capturing the image."
+    },
+    "image_name": {
+      "Value": "windows-server-81-${ID}",
+      "Description": "The name of the translated image."
+    },
+    "family": {
+      "Value": "",
+      "Description": "Optional family to set for the translated image"
+    },
+    "description": {
+      "Value": "",
+      "Description": "Optional description to set for the translated image"
+    },
+    "import_network": {
+      "Value": "global/networks/default",
+      "Description": "Network to use for the import instance"
+    },
+    "import_subnet": {
+      "Value": "",
+      "Description": "SubNetwork to use for the import instance"
+    }
+  },
+  "Steps": {
+    "translate-image": {
+      "IncludeWorkflow": {
+        "Path": "./translate_windows_wf.json",
+        "Vars": {
+          "source_disk": "${source_disk}",
+          "install_gce_packages": "${install_gce_packages}",
+          "sysprep": "${sysprep}",
+          "drivers": "gs://gce-windows-drivers-public/release/win6.3/",
+          "version": "6.3",
+          "task_reg": "./task_reg_2012r2",
+          "task_xml": "./task_xml",
+          "byol": "true",
+          "import_network": "${import_network}",
+          "import_subnet": "${import_subnet}"
+        }
+      },
+      "Timeout": "60m"
+    },
+    "create-image": {
+      "CreateImages": [
+        {
+          "Name": "${image_name}",
+          "SourceDisk": "${source_disk}",
+          "Licenses": ["projects/windows-cloud/global/licenses/windows-server-2012-r2-byol"],
```

Comment:
Do we not have a windows-8-1-enterprise-byol license?

Reply:
This was probably an oversight, I will update and create the licence if it does not exist.

Reply:
I am moving the Windows 8.1 import workflow into a separate PR.